tests/src/test/scala/org/http4s/DecodeSpec.scala

```diff
@@ -95,50 +95,38 @@ class DecodeSpec extends Http4sSuite {
       // Not a valid first byte in UTF-8
       val source = Stream(0x80.toByte)
       val decoded = source.through(decode[Fallible](Charset.`UTF-8`)).compile.string
-      assert(decoded match {
-        case Left(_: MalformedInputException) => true
-        case _ => false
-      })
+      val Left(_: MalformedInputException) = decoded
```

Comment:
This no longer has any assertion - what would a failure look like?

Reply:
If matching fails the failure is `scala.MatchError: Right(_)(of class scala.util.Right)`.